src/ex6/client/src/components/todos-list.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { TodoElement } from "./todo-element.js";
+import PropTypes from 'prop-types';
+
+export function TodosList(props) {
+
```

Comment:
use destructor instead of directly using props
ex: const { todos } = props

Reply:
done